cwms-data-api/src/main/java/cwms/cda/data/dto/Blob.java

```diff
@@ -59,5 +60,15 @@ public String toString(){
 
 	@Override
 	public void validate() throws FieldException {
+		if (getOfficeId() == null) {
+			throw new FieldException("An officeId is required when creating a blob");
```

Comment:
Is it me, or should this constructor take a field name.

... or maybe a builder that allows all possible details to be entered before throwing.

Nothing for you to fix, just pontificating.

Reply:
we may eventually get annoyed enough to come up with an annotation or cleaner way of doing this